lago/subnet_lease.py

```diff
@@ -207,9 +207,8 @@ def _lease_valid(self, lease):
         if not lease.exist:
             return None
 
-        env = lease.has_env
-        if env:
-            return env
+        if lease.has_env:
```

Comment:
Not what I had in mind - in `Lease` class I changed 'has_env' to return a Boolean variable, which I think is what you'd expect from its name(has_env).
I think the performance issue here is probably neglectable, but I could:
change `has_env -> get_env` - and make it return the env or `None`, what do you think?